### PR TITLE
feat/ST-995 Created Edit cica case details in the drop down to start event

### PR DESCRIPTION
**Pull request by muradalamv1**
### Change description ###

Created Edit cica case details in the drop down to start event

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/sptribs/caseworker/model/EditCicaCaseDetails.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+import uk.gov.hmcts.sptribs.ciccase.model.access.CaseworkerAndSuperUserAccess;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+public class EditCicaCaseDetails {
+
+    @CCD(
+        label = "nn",
```
Comment (sjamihmcts) [inline src/main/java/uk/gov/hmcts/sptribs/caseworker/model/EditCicaCaseDetails.java:17]: Do we need the label ?

File: src/main/java/uk/gov/hmcts/sptribs/ciccase/model/CaseData.java [hunk 2]
```diff
@@ -50,13 +51,20 @@
 public class CaseData {
 
 
-    @JsonUnwrapped(prefix = "orderContent")
```
Comment (sjamihmcts) [inline src/main/java/uk/gov/hmcts/sptribs/ciccase/model/CaseData.java:53]: Seems like you deleted the prefix for field defined in line 65

File: src/main/java/uk/gov/hmcts/sptribs/caseworker/model/EditCicaCaseDetails.java (hunk 1 above)
Comment (muradalamv1) [reply src/main/java/uk/gov/hmcts/sptribs/caseworker/model/EditCicaCaseDetails.java:17]: this has changed now.

File: src/main/java/uk/gov/hmcts/sptribs/ciccase/model/CaseData.java (hunk 2 above)
Comment (muradalamv1) [reply src/main/java/uk/gov/hmcts/sptribs/ciccase/model/CaseData.java:53]: all fixed

File: src/test/java/uk/gov/hmcts/sptribs/testutil/TestEventConstants.java [hunk 3]
```diff
@@ -11,6 +11,7 @@ public final class TestEventConstants {
     public static final String CASEWORKER_CLOSE_THE_CASE = "caseworker-close-the-case";
     public static final String CASEWORKER_CONTACT_PARTIES = "contact-parties";
     public static final String CASEWORKER_CREATE_DRAFT_ORDER = "create-draft-order";
+    public static final String CASEWORKER_EDIT_CICA_CASE_DETAILS = "caseworker-eidt-cica-case-details";
```
Comment (sjamihmcts) [inline src/test/java/uk/gov/hmcts/sptribs/testutil/TestEventConstants.java:14]: Typo error in value.

**Pull request merged**